solutions.txt

```diff
@@ -225,233 +212,4 @@ function myFunction(item) {
 }
 
```

Comment:
a couple of ways to challenge yourself 😄 
* try using nested loops to get each inner array, and then each element inside the array
* try using `reduce()`